assets/js/form.js

```diff
@@ -0,0 +1,43 @@
+// whole form
+// view
+var FormView = Backbone.View.extend({
+  template: AppTemplates.form,
+
+  el: '#target',
+
+  initialize: function() {
+    this.render();
+  },
+
+  events: {
+    'click .submit': 'submitForm'
```

Comment:
Instead of listening for the `.submit` button to fire `click`, listen for a `form` to first `submit`. If a user hit's the `ENTER` key they will end up losing all of their progress.

Instead a form will submit either when a user hits `ENTER` on their keyboard or if any button is clicked within the form.

You will also want to look at `ev.preventDefault` in jQuery or else you won't be able to stop the form from going through and refreshing the page. http://api.jquery.com/event.preventdefault/